bears/rust/RustClippyLintBear.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.misc.Shell import run_shell_command
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.CargoRequirement import (
+        CargoRequirement)
+
+
+@linter(executable='cargo',
+        output_format=None,
+        global_bear=True,
+        use_stderr=True)
```

Comment:
do you also need stdout? If not you should deactivate it with `use_stdout=False` (it's not automatically deactivated with `use_stderr=True`)